src/main/java/dev/morling/jfrunit/JfrEvents.java

```diff
@@ -122,14 +125,18 @@ public void reset() {
     }
 
     public Stream<RecordedEvent> getEvents() {
-        return events.stream();
+        return stream();
     }
 
     public Stream<RecordedEvent> filter(Predicate<RecordedEvent> predicate) {
-        return events.stream().filter(predicate);
+        return stream().filter(predicate);
     }
 
     public Stream<RecordedEvent> stream() {
```

Comment:
What's the motivation for exposing this in addition to `getEvents()`?

Reply:
For the `*Spec` I've always seen it like that with Spock. Sometimes people use `*Specification`. I've never saw `*Test`. Also Spock's methods suggest a spec by having `setupSpec()` and `cleanupSpec()`. But I can live with anything you would like :)

> What's the motivation for exposing this in addition to getEvents()?

Reply:
> For the `*Spec` I've always seen it like that with Spock.

Ok, cool, let's follow this pattern then.

> I don't think I'm exposing anything new.

Hum, yes, you're right, it was like that before. Which raises the question: should we make `stream()` private, and perhaps rename `getEvents()` into `events()` (or `recordedEvents()`), for more consistency and exposing only a single public method for the purpose of getting the events?